src/schema.php

```diff
@@ -33,6 +33,10 @@ public function add_hooks() {
 			\add_filter( 'wpseo_debug_json_data', [ $this, 'replace_domain' ] );
 		}
 
+		if ( $this->option->get( 'enable_structured_data_blocks' ) === true ) {
+			\add_filter( 'wpseo_debug_json_data', [ $this, 'enable_feature_flag' ] );
```

Comment:
I am not sure if this is the right hook. I think `init` would be a more appropriate one.

Reply:
Done